src/Graphics/Instance.cpp

```diff
@@ -85,16 +85,23 @@ Instance::Instance(App* app) :
     m_EnabledInstanceExtensions.reset(static_cast<size_t>(VulkanInstanceExtension::kVK_EXT_debug_utils));
 #endif
 
-    std::vector<const char*> enabledInstanceExtensions = BuildEnabledExtensions<VulkanInstanceExtensionBitset, VulkanInstanceExtension>(instanceExtensionProperties,
-                                                                                                                                        m_EnabledInstanceExtensions);
-
     // Instance Creation
 #ifdef VK_API_VERSION_1_1
     m_ApiVersion = m_Context.enumerateInstanceVersion(); // TODO: potential crash
 #else
     m_ApiVersion = VK_API_VERSION_1_0;
 #endif
 
+    uint32_t minorVersion = VK_API_VERSION_MINOR(m_ApiVersion);
+    uint32_t majorVersion = VK_API_VERSION_MAJOR(m_ApiVersion);
+    if (majorVersion >=1 && minorVersion >= 3)
```

Comment:
You can directly do m_ApiVersion >= VK_API_VERSION_1_3, and also this code seems to be redundant as the m_ApiVersion is already as high as possible

Reply:
Deleted!